README.md

```diff
@@ -2,11 +2,36 @@
 
 NOTE: The project is under development.
 
+## PREREQUISITES
 
-## Deploy Onload Operator
+1) Go version 1.21+
+2) The binaries of oc, kubectl present in $PATH
+3) Docker or a symbolic link to a API equivalent (e.g. podman) present in $PATH.
+4) The following exports.
+```text
+export KUBECONFIG=<Path to Kubeconfig for your deployment>
+export REGISTRY_LOCATION=<URL for your docker registry>
+export REGISTRY_PORT=$REGISTRY_PORT
```

Comment:
I would remove `REGISTRY_PORT` unless it's used elsewhere alone without `REGISTRY_URL`.